client/src/components/App/index.js

```diff
@@ -1,4 +1,11 @@
 import React from 'react';
+import DetailsPage from '../DetailsPage';
 import './style.css';
 
-export default () => <h1>Hello World</h1>;
+export default () => {
+  return (
+    <div className="pageDetalis">
```

Comment:
It doesn't really matter as long as the class name is the same everywhere, but this should be 'pageDetails'